src/Locale.php

```diff
@@ -56,8 +56,8 @@ final class Locale implements MiddlewareInterface
      * @param string[] $ignoredRequestUrlPatterns {@see WildcardPattern Patterns} for ignoring requests with URLs matching.
      * @param ?DateInterval $cookieDuration Locale cookie lifetime. Effective only when {@see $saveLocale} is set to
      * `true`. `null` disables saving locale to cookies completely.
-     * @param bool $secureCookie Whether middleware should flag locale cookie as "secure". Effective only when
-     * {@see $saveLocale} is set to `true` and {@see $cookieDuration} is not `null`.
+     * @param bool $secureCookie Whether middleware should flag locale cookie as "secure." Effective only when
```

Comment:
Why put dot inside the quotes?

Reply:
Grammar. [See "Place punctuation marks inside quotation marks."](https://www.grammarly.com/blog/quotation-marks/)

Reply:
All examples there refer to direct speech, while here it's not used. Are you sure it should be applied here as well?

Reply:
Yes.

Reply:
Could you provide an example too?